src/clj/reviews_next/routes.clj

```diff
@@ -1,8 +1,18 @@
 (ns reviews-next.routes
   (:require [bidi.ring :as bidi]
-            [reviews-next.handlers.pages :as pages]))
+            [ring.adapter.jetty :as jetty]
+            [clojure.pprint :as pp]
+            [reviews-next.handlers.pages :as pages]
+            [ring.middleware.cors :refer [wrap-cors]]
+            [ring.middleware.json :refer [wrap-json-params]]
+            [reviews-next.handlers.review-event :as review-event]))
+
+(def handler
+  (wrap-cors routes :access-control-allow-origin [#"http://localhost:3000/"]
+                       :access-control-allow-methods [:get :put :post :delete]))
 
 (def routes
   ["/" {"" pages/index
+        "api/" {"review-event" {:post {"" (wrap-json-params review-event/index)}}}
```

Comment:
If we add new APIs that also require the `wrap-json-params` middleware we will require to add this again and again. Can we place this middleware some place else, so that will works for multiple APIs

This can be taken up in the future when we add new API. Just food of thought.

Reply:
How about if we use partial here? Is this the right usage of it?